recipes/panzi_portable_endian/all/conanfile.py

```diff
@@ -0,0 +1,37 @@
+import os
+from conans import ConanFile, tools
+
+
+class PanziPortableEndian(ConanFile):
+    name = "panzi_portable_endian"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://gist.github.com/panzi/6856583"
+    description = "This provides the endian conversion functions form endian.h on Windows, Linux, *BSD, and Mac OS X"
+    topics = ("conan", "endian")
+    license = "Public Domain"
```

Comment:
🟨 For CCI Team, we have a bunch of these https://github.com/conan-io/conan-center-index/search?p=2&q=Public+Domain... but it does not match the SPDX identifiers... Recently we use "unlicensed" is that our rule going forward?

Reply:
That is a good idea. Fixed

Reply:
[Unilicense](https://spdx.org/licenses/Unlicense.html) is SDPX id for public domain, let's use it going forward
see also https://github.com/conan-io/conan-center-index/pull/4859#discussion_r593639688